docs/configuration_options.md

```diff
@@ -0,0 +1,275 @@
+# Configuration Reference
+
+<!-- Sub:[TOC] -->
+
+## General Settings
+
+### access_token
+
+**Required**
+
+Sets the access token used to send payloads to rollbar.
+
+Items sent through a given access token arrive in that access token's project
+and respect the rate limits set on that access token.
+
+### async_handler
+
+If `config.use_async = true` explicitly sets the function used to send
+asynchronous payloads to Rollbar. Should be an object that responds to `call`
+Not needed if using one of the built in async reporters:
+
+ * girl_friday
+ * sucker_punch
+ * Sidekiq
+ * Resque
+ * threading
+
+### branch
+
+**Default** "master"
+
+Name of the checked-out source control branch.
+
+### code_version
+
+A string, up to 40 characters, describing the version of the application code.
+
+Rollbar understands these formats:
+
+ * semantic version (i.e. "2.1.12")
+ * integer (i.e. "45")
+ * git SHA (i.e. "3da541559918a808c2402bba5012f6c60b27661c")
+
+### custom_data_method
+
+The method to call to gather custom data to send with each rollbar request.
+
+```ruby
+def custom_data
+  return {
+    :custom => {
+      :key1 => get_key_one,
+      :key2 => get_key_two
+    },
+    :server => {
+      :root => '/home/username/www/'
+    }
+  }
+end
+```
+
+### default_logger
+
+**Default** `Logger.new(STDERR)`
+
+What logger to use for printing debugging and informational messages during
+operation.
+
+### disable_monkey_patch
+
+**Default** `false`
+
+Disables monkey patching all non-core monkey patches. If you do this you'll
+need to manually `use` an appropriate Rollbar middleware.
```

Comment:
Do you think this it's well explained? It's true that `use` is the method used in rack based frameworks but may not be for other libaries/frameworks.

I'd just explain that we disable all the monkey patches and auto-reporting.